### PR TITLE
[GH-26]  Add some basic documentation about message preview bot commands

### PR DESCRIPTION
**Pull request by vespian**
#### Summary

Add some basic documentation about message preview bot commands - follow-up for https://github.com/mattermost/mattermost-plugin-welcomebot/pull/29#issuecomment-587033993

#### Ticket Link

  Fixes https://github.com/mattermost/mattermost-plugin-welcomebot/issues/26

### CODE REVIEW

**Comment by jasonblais** [comment]
@aaronrothschild friendly reminder to help with PM review, thanks!

**Comment by vespian** [comment]
@aaronrothschild Ping.

File: README.md [hunk 1]
```diff
@@ -66,6 +66,11 @@ where
     - **ActionSuccessfulMessage**: Message posted after the user takes this action and joins the specified channels.
     - **ChannelsAddedTo**: List of channel names the user is added to. Must be the channel handle used in the URL, in lowercase. For example, in the following URL the **channel name** value is `my-channel`: https://example.com/my-team/channels/my-channel
 
+The preview of the configured messages can be done via bot commands:
+* `/welcomebot help` - show a short usage information
+* `/welcomebot list` - lists the teams for which greetings were defined
+* `/welcomebot preview` [team-name] - sends ephemeral messages to the user calling the command, with the preview of the welcome message[s] for the given team name and the user that requested the preview
```
Comment (hanzei) [inline README.md]: Should `[team-name]` be part of the code block?
```suggestion
* `/welcomebot preview [team-name]` - sends ephemeral messages to the user calling the command, with the preview of the welcome message[s] for the given team name and the user that requested the preview
```

**Comment by jfrerich** [comment]
@aaronrothschild ping

**Comment by DHaussermann** [comment]
@hanzei I'm unable to remove labels in this repo, Can you please add the `Reviews Complete` label

**Pull request merged**